README.md

```diff
@@ -62,12 +50,10 @@ Feel free to check the [issues page](../../issues/).
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
 ## 📝 License
 
 This project is [MIT](./MIT.md) licensed.
+
+## Aknowledgements
+
+Thanks to Nelson Sakwa, author of the original design.
```

Comment:
- [ ] (Optional) You can add the link to the design so that when people see your project, they can have access to the design you used as a template. 👍